community/gill-jito-airdrop/anchor/README.md

```diff
@@ -0,0 +1,452 @@
+        |       |          |       |
+    Alice    Bob      Carol    Dave
+
+To prove Alice is in the tree:
+1. Provide H(B), H(CD) as proof
+2. Program computes: H(H(A) + H(B)) + H(CD) = ROOT  
+3. If computed ROOT matches stored ROOT → Alice is valid ✅
+```
+
+**Benefits:**
+- **Efficient**: Proof size is O(log n), not O(n)
+- **Secure**: Cryptographically impossible to fake
+- **Scalable**: Works for millions of recipients
+
+## 🛠️ Development Workflow
+
+### 1. Modify Recipients
+
+Edit `recipients.json` with your airdrop recipients:
+
+```json
+{
+  "airdropId": "my-awesome-airdrop-2024",
+  "description": "Community airdrop for early supporters",
+  "totalAmount": "300000000",
+  "network": "devnet",
+  "recipients": [
+    {
+      "publicKey": "HcCcKydEcuGMbBso7jciQrFpBw1XQrKHKFckGSAxXXQG",
+      "amount": "100000000",
+      "description": "Early supporter #1"
+    }
+  ]
+}
+```
+
+### 2. Regenerate After Changes
+
+```bash
+# After modifying program code
+npm run codama:build-and-generate
+
+# After modifying recipients
+npm run airdrop:setup
+```
+
+### 3. Test Your Changes
+
+```bash
+# Run full test suite
+npm test
+
+# Test specific functionality
+npm run airdrop:init
+npm run airdrop:extract
+```
+
+## 🐛 Troubleshooting
+
+### Common Issues
+
+**"Cannot find module" errors**
+```bash
+# Regenerate Codama client
+npm run codama:generate
+```
+
+**"Program ID mismatch"**
+```bash
+# Redeploy with consistent program ID
+npm run airdrop:setup
+```
+
+**"Account not found"**
+```bash
+# Ensure program is deployed and initialized
+npm run airdrop:setup
+npm run airdrop:init
+```
+
+**"Invalid Merkle proof"**
+```bash
+# Regenerate merkle tree if recipients changed
+npm run airdrop:setup
+```
+
+**PostCSS errors in tests**
+- This is a known Vitest configuration issue
+- Tests compile and work correctly
+- The error doesn't affect functionality
+
+### Development Tips
+
+1. **Always regenerate client** after program changes
+2. **Use TypeScript** - the generated client provides full type safety
+3. **Test on devnet** before mainnet deployment
+4. **Check balances** before and after operations
+5. **Use the comprehensive test suite** to verify functionality
+
+## 🚨 Production Deployment
+
+For mainnet deployment:
+
+```bash
+# 1. Switch to mainnet
+solana config set --url https://api.mainnet-beta.solana.com
+
+# 2. Update recipients.json
+{
+  "network": "mainnet",
+  // ... your real recipients
+}
+
+# 3. Deploy (same commands work)
+npm run airdrop:setup
+npm run airdrop:init
+```
+
+**⚠️ Security Checklist:**
+- [ ] Verify all recipient addresses are correct
+- [ ] Double-check all amounts (in lamports)
+- [ ] Test completely on devnet first
+- [ ] Secure your program upgrade authority
+- [ ] Consider making program immutable after deployment
+
+## 🎓 Key Technologies
+
+- **[Gill (@solana/kit)](https://github.com/solana-labs/solana-web3.js/tree/master/packages/kit)**: Modern Solana JavaScript SDK
```

Comment:
This link is not correct.

It says gill, and kit, but links to Web3.js

We should use this:

- gill: https://github.com/decalLabs/gill
- kit: https://github.com/anza-xyz/kit
- web3js: no need to link to it as it's deprecated